src/python/pants/engine/internals/native_engine.pyi

```diff
@@ -13,6 +13,15 @@ class PyDigest:
     @property
     def serialized_bytes_length(self) -> int: ...
 
+class PySnapshot:
+    def __init__(self) -> None: ...
```

Comment:
This is surprising to me - is it no longer possible to use the constructor with 3 args?

Reply:
It is ~impossible to create a correct `Snapshot` other than "the empty snapshot" by hand, because you need assistance computing the correct `Digest` for the files. Consequently, that was the only call to the `Snapshot` constructor on the Python side.